source/runner/templates/bitbucketServerTemplate.ts

```diff
@@ -76,6 +65,8 @@ export const messageForResultWithIssues = `${warningEmoji} Danger found some iss
  */
 export function template(dangerID: string, results: DangerResults, commitID?: string): string {
   return `
+${messageForResultWithIssues}
```

Comment:
Wouldn't this be added to the danger comment? If yes, why?

Reply:
In bitbucket cloud implementation --->[Bitbucket Cloud Template](https://github.com/danger/danger-js/blob/4264b34c50f4e386524b441f42b64c7376de4c43/source/runner/templates/bitbucketCloudTemplate.ts#L56). 

```javascript
export function template(dangerID: string, results: DangerResults, commitID?: string): string {
  let summaryMessage: string
  if (!results.fails.length && !results.warnings.length) {
    summaryMessage = `${successEmoji}  All green. ${compliment()}`
  } else {
    summaryMessage = messageForResultWithIssues
  }
  return `
  ${summaryMessage}
   .....
```

The summary message will be inserted to every template. Maybe bitbucket server also need this? If so, i will extract this message hint to a independent function.

Reply:
I feel we shodn't do that. 
Bitbucket server is used in a lot of repos for years now with this template, and I personally don't see any benefit that this message is giving to the template that could justify change it.
For example I use the bitbucket template, and I have my dangerfile divided in tasks and every taks makes a report... I've sent some times to try to not make the danger comment a little bit smaller, because as I said I had a lot of reports and the comment could become really big given there was also the coverage of the modified files... This would make my comment bigger, but the information I'm getting in exchange would not be really useful IMO

Reply:
Bitbucket cloud is a new implementation, and I remember there was a reason for that message, and didn't modify any existing template, then I suppose in that case is different and totally ok

Reply:
Okay! I will revert the bitbucket server template change. By the way,  is there a guide or a effective way for developer to debug locally?